spiceaidocs/docs/reference/spicepod/datasets.md

```diff
@@ -144,6 +144,23 @@ Spice emits a warning if the `time_column` from the data source is incompatible
 :::warning[Limitations]
 
 - String-based columns are assumed to be ISO8601 format.
+
+:::
+
+## `invalid_type_action`
+
+Optional. Specifies the action to take when a data type that is not supported by the data connector is encountered.
+
+The following values are supported:
+
+- `error` - Default. Return an error when an unsupported data type is encountered.
+- `warn` - Log a warning and ignore the column containing the unsupported data type.
+- `ignore` - Log nothing and ignore the column containing the unsupported data type.
+
+:::warning[Limitations]
+
+Not all connectors support specifying an `invalid_type_action`. When specified on a connector that does not support the option, the connector will fail to register.
```

Comment:
Since support is so limited at the moment, we should put the list here (just DuckDB for now)